src/api/tests/participantsService.spec.ts

```diff
@@ -0,0 +1,88 @@
+import request, { Request } from 'supertest';
+
+import { participantsRouter } from '../participantsRouter';
+import { ParticipantRequest } from '../services/participantsService';
+import { mockParticipant, mockUser } from './queryMocks';
+import useTestServer, { api } from './utils';
+
+describe('Participant Service Tests', () => {
+  const withToken = useTestServer();
+
+  beforeEach(() => {
```

Comment:
This should be beforeAll - with beforeEach it adds the route over and over so there end up being many identical routes.